README.md

```diff
@@ -1,41 +1,74 @@
-on the contents of `.git_cml/{checkpoint_file_name}`.
-
-For a given parameter group, `git-cml` will store either the group's parameter's
-values or an update to the values in the event that it can be stored more
-efficiently.
-For example, if a sparse update is made to a parameter group, `git-cml` will
-store the sparse update in a sparse format to save storage costs.
-For a given parameter group, all updates made since the last time the group's
-values were stored are stored in an `updates` subdirectory within
-`.git_cml/{checkpoint_file_name}/{parameter_name}`.
-Whenever a dense update is made to a parameter group, the new full set of
-parameter values is stored and all prior updates are removed from the
-`updates` subdirectory.
-All parameters and updates are currently stored via `git-lfs`.
-
-`git diff` will identify which parameter groups have changed and how.
-
-`git merge` will assume that all merges to the checkpoint (i.e. to parameter
-group files) result in merge conflicts and offer various possible automated
-merging strategies that can be tried and vetted.
+# git-theta
+
+git extension for collaborative, continual, and communal model development. 
+
+# How to use this repository?
+## LFS installation 
+Download the LFS package from the [website](https://github.com/git-lfs/git-lfs/releases/tag/v3.2.0). For Linux users, download the amd64 version from the list of assests in the website. 
+
+## Getting started
+clone the repository  
+```bash
+git clone https://github.com/r-three/git-theta.git
+```
+Install the packages by running:
+```bash
+cd git-theta
+pip install -e .
+```
+## Installing git theta
+<!--Is repository same as codebase?-->
+You can initialize the git theta in the home directory of the codebase to track code and models as follows:
+```bash
+git theta install
+```
+
+The following lines will be added to the `.gitconfig` file in the root directory of the user after the successful installation. 
+```
+[filter "lfs"]
+        smudge = git-lfs smudge -- %f
+        required = true
+        clean = git-lfs clean -- %f
+[filter "theta"]
+        clean = git-theta-filter clean %f
+        smudge = git-theta-filter smudge %f
+        required = true
+```
+
+# Example Usage
+<!--Create a folder with a text file and a model checkpoint. Initialize it as a git repository.-->
+First, initialize git in the home directory of the codebase
+```bash
+git init
+```
+In order to track the model checkpoint using git theta, run this command
+```bash 
+git theta track {path_to_model_checkpoint}
+```
+
+The above command adds the following lines to the `.gitattributes` files in the home directory.
+```
+".git_theta/{model_checkpoint_name}/**" filter=lfs diff=lfs merge=lfs -text
+{path_to_model_checkpoint} filter=theta
+```
+It also creates a `.git_theta/{model_checkpoint_name}` folder  in the home directory of the codebase. 
+
+Add the model to git by running the command 
+```bash
+git theta add {path_to_model_checkpoint}
```

Comment:
Unless I was missing something, git-theta seems to currently only work when the models are saved as serialized state_dicts and not when saved as pytorch nn.Modules. We should mention that here in the readme